sdk/tests/wallet/claim_outputs.rs

```diff
@@ -85,25 +75,31 @@ async fn claim_2_basic_outputs_no_outputs_in_claim_account() -> Result<()> {
 
     // Equal to minimum required storage deposit for a basic output
     let micro_amount = 42600;
-    let tx = account_0
-        .send_micro_transaction(
-            vec![
-                AddressWithMicroAmount {
-                    address: account_1.addresses().await?[0].address().to_bech32(),
-                    amount: micro_amount,
-                    return_address: None,
-                    expiration: None,
-                },
-                AddressWithMicroAmount {
-                    address: account_1.addresses().await?[0].address().to_bech32(),
-                    amount: micro_amount,
-                    return_address: None,
-                    expiration: None,
-                },
-            ],
-            None,
-        )
-        .await?;
+
+    let token_supply = account_0.client().get_token_supply().await?;
+    let rent_structure = account_0.client().get_rent_structure().await?;
+    let expiration_time = account_0.client().get_time_checked().await? + 86400; // 1 Day from now
+
+    // Get the output's rent cost
+    let output = BasicOutputBuilder::new_with_amount(micro_amount)?
+        .add_unlock_condition(AddressUnlockCondition::new(
+            *account_1.addresses().await?[0].address().as_ref(),
+        ))
+        .add_unlock_condition(ExpirationUnlockCondition::new(
+            *account_0.addresses().await?[0].address().as_ref(),
+            expiration_time,
+        )?)
+        .finish_output(token_supply)?;
+    let rent_cost = output.rent_cost(&rent_structure);
+
+    // Update the amount to include the rent cost of the expiration condition
+    let output = BasicOutputBuilder::from(output.as_basic())
+        .with_amount(micro_amount + rent_cost)?
+        .finish_output(token_supply)?;
```

Comment:
Just thought about this again, I think
```
    // Equal to minimum required storage deposit for a basic output
    let micro_amount = 42600;
```
can be removed and here this can be just
```suggestion
    let output = BasicOutputBuilder::new_with_minimum_storage_deposit(rent_structure)?
        .add_unlock_condition(AddressUnlockCondition::new(
            *account_1.addresses().await?[0].address().as_ref(),
        ))
        .add_unlock_condition(ExpirationUnlockCondition::new(
            *account_0.addresses().await?[0].address().as_ref(),
            expiration_time,
        )?)
        .finish_output(token_supply)?;
```
because we don't have any storage deposit return, we don't need to add more amount

Reply:
That fn is very useful. I didn't realize it existed.